modules/mobile-agents/android/client/src/org/wso2/mdm/agent/utils/StreamHandler.java

```diff
@@ -0,0 +1,61 @@
+package org.wso2.mdm.agent.utils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import android.util.Log;
+
+public class StreamHandler {
+	
+	
+	/**
+	 * Close a ByteArrayOutputStream passed in.
+	 * @param stream - ByteArrayOutputStream to be closed.
+	 */
+	public static void closeOutputStream(OutputStream stream, String tag){
+		if(stream!=null){
+			try {
+	            stream.close();
+            } catch (IOException e) {
+            	if (Constants.DEBUG_MODE_ENABLED) {
+    				Log.e(tag, "Exception occured when closing ByteArrayOutputStream." + e);
+    			}
+            }
+		}
+	}
+	
+	/**
+	 * Close a InputStream passed in.
+	 * @param stream - InputStream to be closed.
+	 */
+	public static void closeInputStream(InputStream stream, String tag){
+		if(stream!=null){
+			try {
+	            stream.close();
+            } catch (IOException e) {
+            	if (Constants.DEBUG_MODE_ENABLED) { 
+    				Log.e(tag, "Exception occured when closing InputStream."+e);
```

Comment:
Hi Prabath,

I will be resolving these tonight and will push by tomorrow. And by the
way, I'm in the process of writing a maven script so that the Android agent
can also be built in similar manner to all other components. I can complete
that as well for this release.

Thanks

On Mon, Mar 23, 2015 at 11:59 AM, Prabath Abeysekara <
notifications@github.com> wrote:

> In
> modules/mobile-agents/android/client/src/org/wso2/mdm/agent/utils/StreamHandler.java
> <https://github.com/wso2/product-mdm/pull/25#discussion_r26915427>:
>
> > +    			}
> > +            }
> > +		}
> > +	}
> > +	
> > +	/**
> > +	 * Close a InputStream passed in.
> > +	 * @param stream - InputStream to be closed.
> > +	 */
> > +	public static void closeInputStream(InputStream stream, String tag){
> > +		if(stream!=null){
> > +			try {
> > +	            stream.close();
> > +            } catch (IOException e) {
> > +            	if (Constants.DEBUG_MODE_ENABLED) {
> > +    				Log.e(tag, "Exception occured when closing InputStream."+e);
>
> Formatting issues. Please use appropriate spaces between string literals.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/product-mdm/pull/25/files#r26915427>.
>



-- 
Kasun Dananjaya Delgolla

Software Engineer
WSO2 Inc.; http://wso2.com
lean.enterprise.middleware
Tel:  +94 11 214 5345
Fax: +94 11 2145300
Mob: + 94 771 771 015
Blog: http://kddcodingparadise.blogspot.com
Linkedin: *http://lk.linkedin.com/in/kasundananjaya
<http://lk.linkedin.com/in/kasundananjaya>*